mysql-test/suite/innodb/r/discard_tablespace.result

```diff
@@ -1,4 +1,31 @@
 #
+# PS-7865: Dropping a table with discarded tablespace crashes the server
+#
+CREATE TABLE t1(a INT)ENGINE=innodb;
+INSERT INTO t1 VALUES (10);
+SELECT * FROM t1 INTO OUTFILE 'tmp1.txt';
```

Comment:
Why this way? What is the advantage of having 10 separate files? (btw, files should be deleted at the end of the test).
Is it really so much faster with  LOAD DATA INFILE than INSERT INTO... SELECT?

Why not:
```
--let $n=10
--disable_qery_log
while ($n)
{
INSERT INTO t1 SELECT * FROM t1;
--dec $n
}
--enable_query_log
```

or just:
```
--disable_query_log
--let $n=2048
BEGIN;
while ($n)
{
INSERT INTO t1 VALUES (10);
--dec $n
}
COMMIT;
--enable_query_log
```

Reply:
Yes, the last one is also working. Done